scripts/babel-preset-emotion-dev/src/index.js

```diff
@@ -11,8 +11,10 @@ module.exports = () => {
       require.resolve('babel-plugin-fix-dce-for-classes-with-statics'),
       require.resolve('@babel/plugin-transform-flow-strip-types'),
       require.resolve('babel-plugin-codegen'),
-      require.resolve('@babel/plugin-proposal-object-rest-spread'),
```

Comment:
this was not using loose, I've removed it because it's already included nowadays in `@babel/preset-env` which in turn uses `loose` here, so it saves few bytes because we don't duplicate `objectSpread2` helper across 2 packages (they were bundled in as it was a new helper) and we just reuse regular `extends` helper